articles/time-management.md

```diff
@@ -0,0 +1,40 @@
+---
+authorName: Christian Abegg
+authorGithubUsername: abeggchr
+---
+# Start using a time management technique today
+
+So you take a task from the board, work on it from A to Z and on nothing else. Sounds familiar? I mainly know this situation theoretically. In real life there is always some interruption like a review to be conducted, a bug to be analysed or a phone call to be answered. And having the smartphone laying around vibrating with the latest funny pictures on WhatsApp doesn't help either. Even though we use a prioritized list of work items in our teams, my work as a developer is far away from being focussed on a single task. 
+
+Unlike people who perform at their best when being a "firefighter", I do not like this mode of work. But what can you do about it? Most teams use a regular meeting to improve the way how the team works together. Some things can be achieved here, but on a personal level, you will still have to find a way to set the focus on and to deal with interuptions. This is where personal time management techniques step in.
+
+There are a lot of techniques available - I recommend to have a look at the following three:
+* Getting Things Done (GTD)
+* Personal Kanban
+* Pomodoro Technique
```

Comment:
Maybe shorten this paragraph: "I recommmend to try out at least one of the following three techniques" - and then use sub titles for each technique paragraph?